DesktopEdge/Views/ItemRenderers/IdentityItem.xaml

```diff
@@ -29,19 +29,19 @@
                 <Label Name="ToggleStatus" Grid.Column="0" Grid.Row="1" Margin="0,-5,0,0" Content="Enabled" Foreground="White" FontSize="11" Typography.Capitals="AllSmallCaps" HorizontalAlignment="Stretch" HorizontalContentAlignment="Center"></Label>
                 <Rectangle Fill="Transparent" Grid.Column="0" Grid.RowSpan="2" VerticalAlignment="Stretch" HorizontalAlignment="Stretch" Cursor="Hand" MouseUp="ToggledSwitch"></Rectangle>
 
-                <Label Name="IdName" MaxWidth="210" Grid.Row="0" Padding="0,0,26,0" Grid.Column="1" VerticalAlignment="Center" Margin="0,11,0,0" FontFamily="pack://application:,,,/Assets/Fonts/#Open Sans" FontSize="16" Foreground="White" HorizontalAlignment="Left">
-                    <TextBlock TextTrimming="CharacterEllipsis">Identity Name</TextBlock>
+                <Label MaxWidth="210" Grid.Row="0" Padding="0,0,0,0" Grid.Column="1" VerticalAlignment="Center" Margin="0,11,0,0" FontFamily="pack://application:,,,/Assets/Fonts/#Open Sans" FontSize="16" Foreground="White" HorizontalAlignment="Left">
```

Comment:
This bug fix addresses an issue where the text block containing the id name and url was not being properly assigned the "ellipses" effect when it overflowed its boundaries.  The text block is the required referenced point, thus removed the name from label and assigned it to the text block allowing the code to affect it correctly.
![image](https://github.com/user-attachments/assets/2cba0cd9-32b3-4ca2-b14f-ccb1eb744a03)